web/packages/selfhosted/test/polyfill/classic_frames_injected/test.ts

```diff
@@ -13,11 +13,28 @@ describe("Flash inside frame with injected ruffle", () => {
     it("polyfills inside a frame", async () => {
         await injectRuffleAndWait(browser);
         await browser.switchToFrame(await browser.$("#test-frame"));
-        await browser.$("<ruffle-object />").waitForExist();
+        // TODO: After https://github.com/webdriverio/webdriverio/issues/13218 is fixed, use browser.$("<ruffle-object />")
+        await browser
+            .$(
+                () =>
+                    (
+                        document.querySelector(
+                            "#test-frame",
+                        ) as HTMLIFrameElement
+                    ).contentDocument?.body.querySelector(
+                        "ruffle-object",
+                    ) as HTMLElement,
+            )
+            .waitForExist();
```

Comment:
@danielhjacobs Do you think we could hide all of these in a single function ion `utils.ts`, with a TODO comment to inline it back once the linked issue is fixed? :thinking:
Then it may be mergeable even...?

Reply:
Probably. Untested, but I guess this may work.

```ts
export async function waitForRuffleObjectInFrame(browser: WebdriverIO.Browser) {
    await browser
            .$(
                () =>
                    (
                        document.querySelector(
                            "#test-frame",
                        ) as HTMLIFrameElement | HTMLFrameElement
                    ).contentDocument?.body.querySelector(
                        "ruffle-object",
                    ) as HTMLElement,
            )
            .waitForExist();
}
```

Reply:
It did work indeed, thank you!